examples/tutorials/00_api_basics.ipynb

```diff
@@ -998,10 +1036,11 @@
     {
      "data": {
       "text/plain": [
-       "['demo_pipeline2_requirements.txt',\n",
+       "['demo_pipeline2.py',\n",
        " 'demo_pipeline2_dag.py',\n",
-       " 'demo_pipeline2.py',\n",
-       " 'demo_pipeline2_Dockerfile']"
+       " 'demo_pipeline2_Dockerfile',\n",
+       " 'demo_pipeline2_requirements.txt',\n",
+       " '__pycache__']"
```

Comment:
did any of our change cause this rearrangement?

Reply:
https://stackoverflow.com/questions/31534583/is-os-listdir-deterministic said `os.listdir` is in arbitrary order. I guess the order is machine dependent.